modules/CameraCapture/CameraCapture.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file   AreaACameraAction.cpp
+ * @brief  カメラを制御するクラス
+ * @author Hara1274
+ */
+
+#ifndef CAMERACAPTURE_H
+#define CAMERACAPTURE_H
```

Comment:
```suggestion
#ifndef CAMERA_CAPTURE_H
#define CAMERA_CAPTURE_H
```
細かいけど、単語と単語の間もアンダースコア入れてほしい。他のも入るはずだから。
前年のままだけど、前年のが入ってないのが悪い。申し訳ない🙇‍♂️